### DISCUSSION TITLE
schemaForm中级联组件异步加载每一级的数据

### DISCUSSION THREAD

**Discussion by zkylearner** [alibaba/formily#1330] [Q&A 2.x] [answered]
如题，原来是级联loadData事件处理的，现在在formilyV2 schema表单下要怎么写
fusion cascader-select文档 https://fusion.design/pc/component/cascader-select?themeid=2#dynamic-container

**Comment by janryWang** [comment, marked as answer]
https://fusion.formilyjs.org/components/cascader 这里的例子全是异步数据的例子
